contracts/DLT/TestDLT.sol

```diff
@@ -3,9 +3,14 @@ pragma solidity 0.8.17;
 
 import { DLT } from "./DLT.sol";
 import { DLTEnumerable } from "./extensions/DLTEnumerable.sol";
+import { DLTPermit } from "./extensions/DLTPermit.sol";
 
-contract TestDLT is DLTEnumerable {
-    constructor(string memory name, string memory symbol) DLT(name, symbol) {
+contract TestDLT is DLT, DLTEnumerable, DLTPermit {
+    constructor(
+        string memory name,
+        string memory symbol,
+        string memory version
+    ) DLT(name, symbol) DLTPermit(name, version) {
         name;
         symbol;
```

Comment:
maybe it's time use name and symbol or to remove them

Reply:
What do you mean by using?